Permission.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Permission"
-  s.version      = "2.0.3"
+  s.version      = "2.1.0"
```

Comment:
I prefer to change it just before pushing it (and it's dependent on the git tag)